maidsafe/safe_browser
# fix(loading-logic): able to move backwards to about:blank or reload about:blank

Resolves #434 

QA:
- Open new tab
- Navigate to any URL, except than `http`
- Either go backwards or type `about:blank` into address bar
- Observe blank page loaded into webview, as opposed to previous `webview` content remaining
